Tests/Editor/GameLiftPlugin/AnywherePageTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement;
+using AmazonGameLiftPlugin.Core.SettingsManagement.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace Editor.Window.UnitTests
+{
+    [TestFixture]
+    public class AnywherePageTests
+    {
+
+        private Mock<GameLiftPlugin> _gameLiftConfigMock;
+        private Mock<IAmazonGameLiftClientWrapper> _gameLiftWrapperMock;
+        private Mock<IAwsCredentialsFactory> _awsCredentialsFactory;
+        private Mock<CoreApi> _coreApi;
+        private Mock<IAmazonGameLiftClientFactory> _amazonGameLiftClientFactory;
+        
+        [SetUp]
+        public void Setup()
+        {
+            _gameLiftConfigMock = new Mock<GameLiftPlugin>();
+            _gameLiftWrapperMock = new Mock<IAmazonGameLiftClientWrapper>();
+            _awsCredentialsFactory = new Mock<IAwsCredentialsFactory>(); 
+            _coreApi  = new Mock<CoreApi>();
+            _amazonGameLiftClientFactory = new Mock<IAmazonGameLiftClientFactory>();
+        }
+        
+        private readonly TextProvider _textProvider = TextProviderFactory.Create();
+
+        private AwsCredentials GetAwsCredentialsWithStubComponents(CoreApi coreApi)
+        {
+            var regionMock = new Mock<RegionBootstrap>(coreApi);
+            regionMock.Setup(target => target.Refresh())
+                .Verifiable();
+
+            var mockLogger = new MockLogger();
+            var creationMock = new Mock<AwsCredentialsCreation>(_textProvider, regionMock.Object, coreApi, mockLogger);
+            creationMock.Setup(target => target.Refresh())
+                .Verifiable();
+
+            var updateMock = new Mock<AwsCredentialsUpdate>(_textProvider, regionMock.Object, coreApi, mockLogger);
+            updateMock.Setup(target => target.Refresh())
+                .Verifiable();
+
+            return new AwsCredentials(creationMock.Object, updateMock.Object, coreApi);
+        }
+
+        private GameLiftRequestAdapter ArrangeAnywhereFleetHappyPath()
+        {
+            var listLocationModel = new List<LocationModel>();
+            listLocationModel.Add(new LocationModel()
+            {
+                LocationName = "custom-location-1"
+            });
+
+            _coreApi.Setup(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>())).Returns( AmazonGameLiftPlugin.Core.Shared.Response.Ok<PutSettingResponse>(new PutSettingResponse()));
```

Comment:
Consider importing a portion of this namespace so we don't need the full reference (ex: import up to Shared)